ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -342,7 +358,8 @@ class CommandTrackerFlowTest
         val Handle(submissions, results, _, completionStreamMock) =
           runCommandTrackingFlow(allSubmissionsSuccessful)
         val timedOutCommandId = "timedOutCommandId"
-        val submitRequestShortDedupTime = newSubmission(timedOutCommandId, Some(shortDuration))
+        val submitRequestShortDedupTime =
```

Comment:
We should not be referencing 'deduplication time' here anymore, right?
```suggestion
        val submitRequestShortTimeout =
```

Reply:
I inlined the variable.